modules/understanding-upgrade-channels.adoc

```diff
@@ -7,79 +7,89 @@
+The fast channel, containing only GA releases, is updated with new
+{product-version} patch versions as soon as Red Hat declares they are generally
+available. Use this channel if you wish to receive updates as soon as they are
+available or for your pre-production environments when participating in the connected
+customer program. This channel will contain all z-stream ({product-version}.z)
+updates but will not suggest upgrades to the next minor release (4.5.z) when the next
+minor release is available. This channel also includes the upgrade from the previous
+4.3 release.
 
 = stable-{product-version}
 
-The stable channel will contain updates on a time delay as they are gradually
-rolled out to customers based on data from our SRE teams, support services, and
-pre-production and production environments that participate in our connected
-customer program, rather than being immediately available as they are in the fast
-channel. For patch and CVE fixes this can range from several hours to a day and
-allows an extra period of assessment in how the software performs. If issues are
-detected during rollout, upgrades to that version may be blocked in both the fast
-and stable channels, and a new version may be introduced that will be the new
-preferred upgrade target.
+The stable channel, containing only GA releases, will contain updates on a time delay
+as they are gradually rolled out to customers based on data from our SRE teams,
+support services, and pre-production and production environments that participate in
+our connected customer program, rather than being immediately available as they are
+in the fast channel. For patch and CVE fixes this can range from several hours to a
+day and allows an extra period of assessment in how the software performs. This
+channel also includes the upgrade from the previous 4.3 release.
+
+= Fast and Stable Channel Usage and Strategies
+
+The fast-{product-version} and stable-{product-version} channels present a choice between receiving
+GA updates as soon as they are available or allowing Red Hat to control the rollout
+of those updates. If issues are detected during rollout, upgrades to that version may
+be blocked in both the fast and stable channels, and a new version may be introduced
+that will be the new preferred upgrade target.
 
 Customers can improve this process by configuring pre-production systems on the
 fast channel, production systems on the stable channel, and participating in Red
-Hat’s connected customer program - this allows Red Hat to observe the impact of
-updates on your specific hardware and software configurations. Future releases may
+Hat’s connected customer program. This program allows Red Hat to observe the impact
+of updates on your specific hardware and software configurations. Future releases may
 improve or alter the pace at which updates move from the fast to the stable channels.
 
 If issues are discovered with an upgrade between patch levels, Red Hat may withdraw
```

Comment:
Rewording ideas for the *Upgrade Version Paths* section below:

> {product-title} maintains an upgrade recommendation service that understands the version of {product-title} you have installed as well as releases to which you may safely update.  You can imagine seeing the following in the fast-{product-version} channel:
>
> * {product-version}.0
> * {product-version}.1
> * {product-version}.3
> * {product-version}.4
>
> The service only recommends upgrades that have been tested and have no serious issues.  If you are on {product-version}.1 and {product-title} is allowing you to select {product-version}.4, then it is safe for you to go from .{product-version}.1 to .{product-version}.4. Likewise, the absence of {product-version}.2 may be due to a CVE that was fixed in {product-version}.3 and Red Hat no longer suggests upgrading to a known vulnerable version.
>
> Update stability depends on your channel.  The presence of an update recommendation in the candidate-{product-version} channel does not imply that the update is supported.  It means that no serious issues have been found with the update yet, but there may not be significant traffic through the update to suggest stability.  The presence of an update recommendation in the fast-{product-version} or stable-{product-version} channels is a declaration that the update is supported while it is in the channel.  Like releases in the stable-{product-version} channel, update recommendations in the stable-{product-version} channel will be promoted from fast-{product-version} with a delay that can range from can range from several hours to a day.  While releases will never be removed from a channel, update recommendations which exhibit serious issues will be removed from all channels.  Updates initiated after the update recommendation has been removed may not be supported.
>
> Red Hat will eventually provide supported update paths from any supported (fast-{product-version} or stable-{product-version}) release to the latest release in {product-version}.z, although there may be delays while safe paths away from troubled releases are constructed and verified.

Reply:
Some of these information are also present previously, we should not repeat these as it confuses the reader


> Update stability depends on your channel. The presence of an update recommendation in the candidate-{product-version} channel does not imply that the update is supported. It means that no serious issues have been found with the update yet, but there may not be significant traffic through the update to suggest stability. The presence of an update recommendation in the fast-{product-version} or stable-{product-version} channels is a declaration that the update is supported while it is in the channel. Like releases in the stable-{product-version} channel, update recommendations in the stable-{product-version} channel will be promoted from fast-{product-version} with a delay that can range from can range from several hours to a day. While releases will never be removed from a channel, update recommendations which exhibit serious issues will be removed from all channels. Updates initiated after the update recommendation has been removed may not be supported.

Reply:
> Some of these information are also present previously...

I would rather lead with update discussion, and only talk about release nodes incidentally.